sip/sip-003-peer-network.md

```diff
@@ -402,7 +403,7 @@ pub struct HandshakeData {
     /// This peer's public key
     pub node_public_key: Secp256k1PublicKey,
 
-    /// Burn chain block height at which this key will expire
+    /// Burn chain block height at which this handshake will expire
```

Comment:
No, it's the `key`.  The `handshake` is just the message -- there is no notion of a message expiring.  The public key, however, does need to be refreshed periodically.